draft-ietf-ohai-ohttp.md

```diff
@@ -945,6 +967,12 @@ A formal analysis of Oblivious HTTP is in {{OHTTP-ANALYSIS}}.
 
 ## Client Responsibilities {#sec-client}
 
+Clients MUST maintain an explicit allowlist of target resources that the client trusts
```

Comment:
Minor readability point: I would introduce this a little bit, say "Because OHTTP does not allow the client to authenticate the Target Resource (only the Gateway Resource), ..."

Reply:
Applied -- thanks!